webextension/background/deviceInfo.js

```diff
@@ -1,10 +1,13 @@
-/* globals chrome */
+/* globals chrome, catcher */
 
 window.deviceInfo = (function () {
   let manifest = chrome.runtime.getManifest();
 
   let platformInfo = {};
   chrome.runtime.getPlatformInfo(function(info) {
+    if (chrome.runtime.lastError) {
```

Comment:
> getPlatformInfo doesn't actually produce a promise

Are you sure? Looks like it does in release channel FF:

<img width="468" alt="screen shot 2017-03-02 at 7 18 28 pm" src="https://cloud.githubusercontent.com/assets/96396/23536907/326fdeba-ff7d-11e6-949c-dbfe0b9eb37d.png">

At any rate, this change is a nice to have, so it's fine to leave as is.